tasks/configure_new.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: PGBouncer | Create database conf directory
+  file:
+    path: "{{ pgbouncer_conf_dir }}"
+    state: directory
+    recurse: yes
+
+- name: PGBouncer | Configure database conf
+  template:
+    src: ../templates/database.ini.j2
+    dest: "{{ pgbouncer_conf_dir }}/{{ item.name }}.ini"
+  with_items: "{{ pgbouncer_database_aliases }}"
```

Comment:
in new release of ansible 2.2 it's required to have a default input as empty list if using with items.